python/agents/blog-writer/blogger_agent/validation_checkers.py

```
@@ -0,0 +1,46 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import AsyncGenerator
```

Ruff (0.12.2)

Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)